db/db_info_dumper.cc

```diff
@@ -64,7 +64,7 @@ void DumpDBFileSummary(const ImmutableDBOptions& options,
         break;
       case kWalFile:
         if (env->GetFileSize(dbname + "/" + file, &file_size).ok()) {
-          char str[16];
+          char str[21];
           snprintf(str, sizeof(str), "%" PRIu64, file_size);
           wal_info.append(file).append(" size: ").append(str).append(" ; ");
```

Comment:
Thanks @ReaLNeroM for the contribution.
It would be better just to use `std::to_string()`:

```
wal_info.append(file).append(" size: ").append(std::to_string(file_size)).append(" ; ");
```

In reality, we should never have WAL size > 1PB (15 decimal number ~= 1 P) which means huge memory buffer size, and the most of filesystems don't even support 1P large single file. But it still worth to change it to to_string.

Reply:
Sounds good!

I agree with you that WAL >1PB is unlikely. I would argue that since the <1PB assumption is very hard to notice, it's still worth resolving this issue.